packages/imitation_game/lib/README_template.dart

```diff
@@ -0,0 +1,108 @@
+String readmeTemplate = """# Imitation Game
```

Comment:
Ultra nit: just let this be a .md file and read it to string for the `Template` call? Might be easier to read for contributors.

Reply:
I thought about it but reading from disk has some issues i can avoid by embedding the template into the binary.  Unfortunately there doesn't seem to be a way to embed assets in dart that keeps it as a md file.